### PR TITLE
Small extensions & add new dataset

### PR DESCRIPTION
**Pull request by PetrovKP**
1. Returned support of hardware and software for the logging.
2. Extend cuml config for `skin_segmentation` workload
3. Add loader for `codrnanorm` dataset

### CODE REVIEW

File: datasets/loader.py [hunk 1]
```diff
@@ -313,6 +313,31 @@ def covertype(dataset_dir=None):
     return True
 
 
+def codrnanorm(dataset_dir=None):
+    """
+    """
```
Comment (Alexsandruss) [inline datasets/loader.py:318]: Empty comment. Is this place for dataset description?

File: runner.py [hunk 2]
```diff
@@ -106,18 +110,17 @@ def generate_cases(params):
...
                     train_data = dataset["training"]
-                    test_data = dataset["testing"]
-
                     file_train_data_x = train_data["x"]
-                    file_train_data_y = train_data["y"]
-                    file_test_data_x = test_data["x"]
-                    file_test_data_y = test_data["y"]
                     paths = f'--file-X-train {file_train_data_x}'
                     if 'y' in dataset['training'].keys():
+                        file_train_data_y = train_data["y"]
```
Comment (Alexsandruss) [inline runner.py:116]: `file_train_data_y` is used on line 130 and now might be unassigned.
I guess, need to change line 130 to `if not utils.is_exists_files([file_train_data_x]):`

File: datasets/loader.py (hunk 1 above)
Comment (PetrovKP) [reply datasets/loader.py]: Apply

File: runner.py (hunk 2 above)
Comment (PetrovKP) [reply runner.py:116]: Make sense, thanks! Applied

**Review by Alexsandruss — APPROVED** [review]

**Pull request merged**